docs/DeveloperGuide.adoc

```diff
@@ -34,24 +34,23 @@ The *_Architecture Diagram_* given above explains the high-level design of the A
 
```

Comment:
![image](https://user-images.githubusercontent.com/42093424/67396588-710f1400-f5da-11e9-8c44-e148cbca1170.png)
Should we update this as well?

Reply:
Maybe it means for future release?